migration-guides/v0.8->v0.9.md

```diff
@@ -0,0 +1,68 @@
+## Migration Guide
+
+The following guide assumes a valid v0.8.x `config.json` file. All changes listed below will need to be implemented in
+order to migrate to Airnode v0.9.0. This document is written in a way that will preserve existing behaviour with earlier
+Airnode versions.
+
+The document also mentions changes of the user facing services related to Airnode, such as airnode-deployer,
+airnode-admin and more.
+
+### Summary
+
+ois update to 1.1.2 deployer CLI changes
+
+1. `ois[n].oisFormat` updated to "1.1.2"
+
+2. `nodeSettings.nodeVersion` updated to "0.9.0"
+
+3. `airnode-deployer remove-with-deployment-details` now accepts a full `--airnode-address` argument instead of the
+   short airnode address (`--airnode-address-short`)
+
+### Details
+
+1. `ois[n].oisFormat`
+
+Updated to "1.1.2"
+
+```diff
+{
+-  "oisFormat": "1.0.0"
++  "oisFormat": "1.1.2"
```

Comment:
TODO: This might change as per https://api3workspace.slack.com/archives/C02AYRX8D89/p1663737468488609?thread_ts=1662763603.054039&cid=C02AYRX8D89